src/Float.mo

```diff
@@ -1,71 +1,375 @@
-/// 64-bit Floating-point numbers
+/// Double precision (64-bit) floating-point numbers in IEEE 754 representation.
+///
+/// Common floating-point constants and functions.
+///
+/// Notation for special values in the documentation below:
+/// `+inf`: Positive infinity
+/// `-inf`: Negative infinity
+/// `nan`: "not a number" (can have different sign bit values, but `nan != nan` regardless of the sign).
+///
+/// Note:
+/// Floating point numbers have limited precision and operations may inherently result in numerical errors.
+///
+/// Examples of numerical errors:
+///   ```motoko name=initialize
+///   assert(0.1 + 0.1 + 0.1 == 0.3); // Fails!
```

Comment:
```suggestion
///   0.1 + 0.1 + 0.1 == 0.3 // False!
```
Subjectively, removing the `assert` seems to be easier to read on the webpage. Wondering what you think.

Reply:
Thanks for this suggestion. Indeed, much easier to read.